### DISCUSSION TITLE
Where is my profile picture taken from?

### DISCUSSION THREAD

**Discussion by alejandrorangll** [community/community#32845] [Other Feature Feedback, Questions, & Ideas] [answered]
I've never changed or added any picture onto my profile picture, however there's one there. So, I just wanted to know how GitHub got access to that picture, I think I only use that picture in my WhatsApp profile picture.

**Comment by radar-blip** [comment]
You must have added it or else it wouldn't be there. 
Unless someone else had that exact same username in the past, but even then, its highly unlikely their profile picture would remain.

**Reply by alejandrorangll** [reply]
Except I didn’t and it’s impossible that anyone else created an account with the same email and it’s not a random picture, is actually my picture.

**Comment by SharfShiplo** [comment, marked as answer]
GitHub may use Gravatar to fetch people's profile picture. You may have a Gravatar config/account if you used Wordpress in the past.

Basically, on a Gravatar account, an image is associated with an email. Services like GitHub can publicly query Gravatar to get the image associated with an email.

Try to connect on Gravatar (with Wordpress credentials I think) to modify/remove the image.

**Reply by alejandrorangll** [reply]
Thanks! I really appreciate it.

This is exactly what happened:
Some evidence here

![9D617BEE-BE03-41D8-BD9B-ECD3949F047B](https://user-images.githubusercontent.com/113403622/190227056-eae53bd3-9f90-4fac-bbe0-9487d67fa4fe.png)
